src/components/tabController/TabBar.tsx

```diff
@@ -117,7 +117,7 @@ interface Props extends TabControllerBarProps, BaseComponentInjectedProps, Forwa
   children?: ChildProps[] | ChildProps;
 }
 
-const FADER_PROPS = {size: 76};
+const FADER_PROPS = {size: 76, tintColor: Colors.$backgroundElevated};
```

Comment:
I feel like this is not as prominent as the regular scheme, probably because there's less contrast with the text color.
Perhaps this should be `Colors.$backgroundDefault`?

Reply:
Agree :)